pallets/gear/src/schedule.rs

```diff
@@ -761,7 +761,7 @@ impl Default for Limits {
 impl<T: Config> Default for InstructionWeights<T> {
     fn default() -> Self {
         Self {
-            version: 11,
+            version: 1050,
```

Comment:
I think we don't want to have to reinstrument programs after every release. It's pointless if nothing related to weight and performance is introduced.

Reply:
Yep, you're right, so it's not supposed to bump every single time, but I want version to reflect release number when it was updated last time. How do you think?

Reply:
Fine with this.